pkg/console/starter/starter.go

```diff
@@ -108,13 +111,37 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		oauthinformers.WithTweakListOptions(tweakListOptionsForOAuth),
 	)
 
-	recorder := ctx.EventRecorder
+	resourceSyncerInformers := v1helpers.NewKubeInformersForNamespaces(
+		kubeClient,
+		api.TargetNamespace,
```

Comment:
Does the `syncer` need to watch all of these namespaces?  Or just `openshift-config-managed`?